llpc/builder/llpcBuilderContext.cpp

```diff
@@ -59,11 +60,16 @@ class PassRegistry;
 static bool Initialized;
 #endif
 
-// -emit-llvm: emit LLVM bitcode instead of ISA
+// -emit-llvm: emit LLVM assembler instead of ISA
```

Comment:
nit: can we call it 'LLVM IR' or 'LLVM assembly'?

Reply:
Done. I went for "LLVM assembly". I don't think "IR" is a good way to distinguish them. Neither *is* IR, but both are representations of IR, one text, one binary.